src/lib.rs

```diff
@@ -102,13 +113,30 @@ pub struct BlockerResult {
 
 impl From<RustBlockerResult> for BlockerResult {
     fn from(br: RustBlockerResult) -> Self {
+        let mut redirect: Option<String> = None;
+        let mut redirect_type: Option<String> = None;
+        if br.redirect.is_some() {
+            let resource = br.redirect.unwrap();
```

Comment:
Can you refactor this to get rid of the unwrap, and even the mutability? I'm thinking something like
```rust
let (redirect, redirect_type) = if let Some(resource) = br.redirect {
    // Code here...
} else {
    (None, None)
}
```